packages/react/src/patterns/sub-patterns/ContentGroup/ContentGroup.js

```diff
@@ -20,41 +21,47 @@ const { prefix } = settings;
  * @param {object} props props object
  * @param {*} props.children JSX Components
  * @param {string} props.heading  Heading objects
```

Comment:
Heading objects I think is simple a a string, not an object.

Reply:
yeah sorry.